docs/ceph-csi-upgrade.md

```diff
@@ -303,4 +304,13 @@ For each node:
   - The pod deletion causes the pods to be restarted and updated automatically
     on the node.
 
-we have successfully upgrade RBD csi from v1.2.2 to v2.0.0
+we have successfully upgraded RBD csi from v1.2.2 to v2.0.0
+
+##### 5.0 Handling the node reboot issue for CephFS and RBD pvc
```

Comment:
Move this to a common bullet rather than under RBD, as this is for both CephFS and RBD.